src/fides/core/config/__init__.py

```diff
@@ -41,11 +42,30 @@ class FidesConfig(FidesSettings):
     """
 
     # Root Settings
-    test_mode: bool = get_test_mode()
-    is_test_mode: bool = test_mode
-    hot_reloading: bool = getenv("FIDES__HOT_RELOAD", "").lower() == "true"
-    dev_mode: bool = getenv("FIDES__DEV_MODE", "").lower() == "true"
-    oauth_instance: Optional[str] = getenv("FIDES__OAUTH_INSTANCE")
+    test_mode: bool = Field(
+        default=get_test_mode(),
+        description="Whether or not the application is being run in test mode.",
+        exclude=True,
+    )
+    is_test_mode: bool = Field(
+        default=test_mode,
+        description="A deprecated analog of 'test_mode' used in some places.",
+        exclude=True,
+    )
+    hot_reloading: bool = Field(
+        default=getenv("FIDES__HOT_RELOAD", "").lower() == "true",
+        description="Whether or not to enable hot reloading for the webserver.",
+        exclude=True,
+    )
+    dev_mode: bool = Field(
+        default=getenv("FIDES__DEV_MODE", "").lower() == "true",
+        description="Similar to 'test_mode', enables certain features when true.",
+        exclude=True,
+    )
+    oauth_instance: Optional[str] = Field(
+        default=getenv("FIDES__OAUTH_INSTANCE"),
+        description="TODO: Should this be hidden?",
```

Comment:
Help

Reply:
@galvana this is one for you - and I'm afraid I actually don't understand our OAuth callback implementation well enough to comment, which is a blindspot for me

Reply:
Sorry for the late response here! This value doesn't need to be hidden, we use it for local OAuth2 testing to help route callback responses to specific Fides instances.

Reply:
@galvana so if its used for testing, should we be surfacing it to production users? Can you provide a code suggestion for a proper description?